js/main.js

```diff
@@ -6,17 +6,79 @@ function getRandomInteger(min, max) {
   }
   return Math.floor(rand);
 }
-getRandomInteger(1, 3);
 
 
 // Результат: случайное число с плавающей точкой число из диапазона "от...до"
 function getRandomFloat(min, max, fraction) {
   const rand = min + Math.random() * (max - min);
   if( max <= min ) {
-    throw new Error('getRandomInteger — максимальное число меньше или равно минимальному');
+    throw new Error('getRandomFloat — максимальное число меньше или равно минимальному');
   }
   return parseFloat(rand).toFixed(fraction);
 }
 
-getRandomFloat(1.1, 2.3, 2);
+const TYPE_HOUSE = [
+  'palace',
+  'flat',
+  'house',
+  'bungalow',
+  'hotel',
+];
+
+const CHECKIN_CHECKOUT = [
+  '12:00',
+  '13:00',
+  '14:00',
+];
+
+const FACILITIES = [
+  'wifi',
+  'dishwasher',
+  'parking',
+  'washer',
+  'elevator',
+  'conditioner',
+];
+
+const PHOTOS = [
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/duonguyen-8LrGtIxxa4w.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/brandon-hoogenboom-SNxQGWxZQi0.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/claire-rendall-b6kAwr1i0Iw.jpg',
+];
+
+const greatFacilities = new Array(getRandomInteger(0, 5)).fill(null).map(() => FACILITIES);
+
+const SIMILAR_WIZARD_COUNT = 10;
+
+const getRandomArrayElement = (elements) => {
+  return elements[getRandomInteger(0, elements.length - 1)];
+};
+
+const createObject = () => {
+  return {
+    author: {
+      avatar: 'img/avatars/user0'+ getRandomInteger(1, 8) + '.png',
+    },
+    offer: {
+      title: 'Заголовок',
+      address: '{{location.x}}, {{location.y}}',
+      price: getRandomInteger(0, 100000),
+      type: getRandomArrayElement(TYPE_HOUSE),
+      rooms: getRandomInteger(0, 10),
+      guests: getRandomInteger(0, 100),
+      checkin: getRandomArrayElement(CHECKIN_CHECKOUT),
+      checkout: getRandomArrayElement(CHECKIN_CHECKOUT),
+      features: getRandomArrayElement(FACILITIES),
```

Comment:
По заданию тут должен быть массив:
![image](https://user-images.githubusercontent.com/30810655/121310165-bf863900-c90b-11eb-99b2-a63fc5e950ad.png)

Тут может пригодиться [метод slice](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/slice)

Reply:
Так же предлагаю переименовать константу, единообразия ради:
```suggestion
      features: getRandomArrayElement(FEATURES),
```